src/lib/storage/fake_device_factory.rb

```diff
@@ -86,8 +84,9 @@ def initialize(devicegraph)
         super(devicegraph)
         @partitions     = {}
         @disks          = Set.new
-        @first_free_cyl = {}
-        @cyl_count      = {}
+        @free_blob      = nil
+        @free_regions   = []
+        @mbr_gap        = nil
```

Comment:
Hmmm... is ok to have this kind of initializations in the factory constructor? Shouldn't be them executed in every call to `load_yaml_file`? According to the documentation is ok to have something like:
```ruby
factory = FakeDeviceFactory.new(devicegraph)
yaml_files.each do |file|
  factory.load_yaml_file(file)
end
```
Would that work?